.github/workflows/ci.yml

```diff
@@ -30,6 +30,37 @@ jobs:
           context: .
           profile-name: "mpc-image-builder"
           file: deployment/Dockerfile-mpc
+          tags: test_image_tag_ci
+          load: true
+
+      - name: Check Docker image can initialize and start
+        run: |
+          touch /tmp/image-digest.bin
+          # Test container startup - fail if container can't start
+          timeout -k 10 120s docker run --rm \
+          -e MPC_HOME_DIR="/tmp" \
+          -e MPC_ACCOUNT_ID=test_image.near \
+          -e RUST_BACKTRACE="full" \
+          -e RUST_LOG="mpc=debug,info" \
+          -e MPC_SECRET_STORE_KEY=BD399143F5B3126098B0EAA023A0E730 \
+          -e MPC_P2P_PRIVATE_KEY=ed25519:2WBi5gRyJntYA7dCyddiwNV2yNTdr5uZhrP4WX5GNBL5DSVCSR7ESvcXF2DBfY5oPYhzBmHnguPVSXjr6UCi8h1g \
+          -e MPC_ACCOUNT_SK=ed25519:6X3Bnghdf89WjHQfHDFYjW2UeNaNqCQ1AUGxX7zgvbJT4KSoeXzuHEdux6A2jsphTZTmS4SUGQRyYqC2ik3UrMP \
+          -e NEAR_BOOT_NODES=ed25519:ERguu7jQuYk8pxNsRC6FdezvNsegBPva1GRGqjmtD7i2@10.10.10.10:24567 \
```

Comment:
What nodes are these? Could we set dummy values here instead? I assume the purpose of this test is not to have this node connect to another node anyway,